pages/api/auth/[...nextauth].ts

```diff
@@ -90,68 +144,6 @@ export const authOptions = {
       },
     }),
   ],
-};
-
-const createOptions = (req: NextApiRequest): NextAuthOptions => ({
-  ...authOptions,
-  callbacks: {
-    jwt: async ({ token, user }) => {
-      if ((!user || req.url === "/api/auth/session?update") && token.sub) {
-        const updatedUser = await getUserDataById(token.sub);
-        if (!updatedUser) {
-          return token;
-        }
-        const user = {
-          id: updatedUser.Ua_Id,
-          name: updatedUser.Ua_login,
-          email: updatedUser.Ua_Email,
-          cryptoalerts: updatedUser.CryptoAlerts,
-          newsletters: updatedUser.Newsletters,
-          productupdate: updatedUser.ProductUpdate,
-          google_sso: updatedUser.GoogleSSO,
-          email_verified: updatedUser.EmailVerified,
-        };
-
-        token = {
-          ...token,
-          ...user,
-          user,
-        };
-      } else {
-        user && (token.user = user);
-      }
-
-      return token;
-    },
-    session: async ({ session, token }) => {
-      session.user = token.user as any;
-      return session;
-    },
-    signIn: async ({ account, profile }: any) => {
-      if (account.provider === "google") {
-        const id = profile.sub;
-        const existingSSOUser = await getExistingUserByID(id);
-
-        if (existingSSOUser.length) {
-          return true;
-        }
-
-        const existing = await getExistingUserByEmail(profile.email);
-        if (existing.length) {
-          const locale = req.cookies["locale"] || "en";
-          return `/${locale}/login?google_sso_error=1`;
-        }
-
-        await registerSSOUser(
-          id,
-          profile.name,
-          profile.email,
-          account.access_token
-        );
-      }
-      return true;
-    },
-  },
 });
 
 export default async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
While we are fixing sessions, please take a look at this error here. `import/no-anonymous-default-export`